js/gui.js

```diff
@@ -306,20 +306,20 @@ GUI_control.prototype.content_ready = function (callback) {
             });
         });
     });
+    
+new jBox('Modal', {
```

Comment:
I don't think this should be in here. `gui.js` contains generic methods that are common to all tabs, and shouldn't contain tab specific stuff. I'd rather use the `show_modal` method below to create the modal dialog in the code for the tab that it appears in.